src/style.css

```
@@ -1 +1,84 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url(./fonts/Roboto-Medium.ttf);
+  font-style: normal;
+  font-weight: 500;
+}
+
+* {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  color: #000;
+
+  box-sizing: border-box;
```

```suggestion
* {
  box-sizing: border-box;
```
Do you really need to change the font styles and color of ALL elements?
Just set if for body or html and inherit where needed
Leave only box-sizing under * selector